src/helpers/puppeteerDriver.js

```diff
@@ -0,0 +1,51 @@
+import puppeteer from 'puppeteer';
```

Comment:
I think that according to our frontend guidelines files that export a class should share the name of the class.

So this file should be `Driver.js`, and server.js should be `Server.js`.